### PR TITLE
Add explicit netstandard2.0 support

### PR DESCRIPTION
**Pull request by am11**
Also removed Mono specific SLN/csprojs files for unification.

Related to maxmind/MaxMind-DB-Reader-dotnet#39.

### CODE REVIEW

File: MaxMind.GeoIP2.UnitTests/DatabaseReaderTests.cs [hunk 1]
```diff
@@ -77,12 +77,7 @@ public void InvalidCountryMethodForCityDatabase_ExceptionThrown()
             {
                 var exception = Record.Exception(() => reader.Country("10.10.10.10"));
                 Assert.NotNull(exception);
-                Assert.Contains(exception.Message,
-#if !NETCOREAPP1_1
-                    "A GeoIP2-City database cannot be opened with the Country method");
-#else
-                    "A GeoIP2-City database cannot be opened with the given method");
-#endif
+                Assert.Contains("A GeoIP2-City database cannot be opened with the", exception.Message);
```
Comment (am11) [inline MaxMind.GeoIP2.UnitTests/DatabaseReaderTests.cs]: Simplified this assertion by removing the last varying part of the sentence as while netcoreapp2.0 has StackTrace (wasn't the case with STD 1.4 https://github.com/maxmind/GeoIP2-dotnet/blob/b2f4334980357c7269136688f8759002000e1220/MaxMind.GeoIP2/DatabaseReader.cs#L472-L481), JIT has improved the optimization and intermediate methods get effectively inlined and we instead get:

> A GeoIP2-City database cannot be opened with the Exception method

**Comment by oschwald** [comment]
:100: Thanks! This looks great. I am just waiting for the _slow_ Travis macOS builds to finish before merging.

**Comment by am11** [comment]
Seems like the macOS of queue of TravisCI was aborted, I reran it on my fork and finally it finished building the entire matrix https://travis-ci.org/am11/GeoIP2-dotnet/builds/273423769 :)

**Comment by oschwald** [comment]
Thanks!

**Comment by oschwald** [comment]
@am11, I finally got this and the related MaxMind.Db change release. Thanks again!

**Pull request merged**